frontend/src/components/contentDisplay/H5PEditor.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+  <h5p-editor ref="editor" :content-id="contentId" />
+</template>
+
+<script setup lang="ts">
+import { IContentMetadata, IEditorModel } from '@lumieducation/h5p-server';
+import {
+  defineElements,
+  H5PEditorComponent,
+} from '@lumieducation/h5p-webcomponents';
+import { onMounted, Ref, ref } from 'vue';
+
+defineProps({
+  /**
+   * The h5p content to display
+   */
+  contentId: {
+    type: String,
+    required: true,
+  },
+});
+
+const emits = defineEmits([
+  /**
+   * Emitted when the editor is closed
+   */
+  'closed',
+]);
+
+defineElements('h5p-editor');
+const editor: Ref<H5PEditorComponent | null> = ref(null);
+
+onMounted(() => {
+  const h5pEditor = editor.value;
+  if (h5pEditor !== null) {
+    h5pEditor.loadContentCallback = async (
+      contentId: string
+    ): Promise<IEditorModel> => {
+      // TODO
+      throw new Error('Error getting content with id: ' + contentId);
+    };
+
+    h5pEditor.saveContentCallback = async (
+      contentId: string,
+      requestBody: { library: string; params: unknown }
+    ): Promise<{ contentId: string; metadata: IContentMetadata }> => {
+      // TODO
+      console.log(requestBody);
+      throw new Error('Error saving content with id: ' + contentId);
+    };
+
+    h5pEditor.disconnectedCallback = (): void => {
```

Comment:
This even exists? What would we need this for? Cant find it in the official [docs](https://docs.lumi.education/npm-packages/h5p-webcomponents#h5peditorcomponent)

Reply:
This isnt specific to h5p but custom html components. It gets called when the editor gets removed from the dom (in our case closed).

Reply:
But of h5p provides no option to close the editor this should be removed

Reply:
I dont think it does by itself

Reply:
I would suggest deleting it in the h5p branch the when we are sure how i works

Reply:
sounds good